From 05461b0cdcf38595cc9ad22a7003727ab0674475 Mon Sep 17 00:00:00 2001
From: mulo01 <mulo01@dfki.de>
Date: Tue, 16 Jul 2024 13:14:39 +0200
Subject: [PATCH] Use root as file prefix

---
 src/Model.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/Model.cpp b/src/Model.cpp
index 965ccf7..0c68977 100644
--- a/src/Model.cpp
+++ b/src/Model.cpp
@@ -464,6 +464,7 @@ namespace envire
                     // create absolute path
                     boost::filesystem::path filePathAbsolute(urdfFilePathAbsolute);
                     std::string rootFolder = filePathAbsolute.parent_path().generic_string();
+                    config["filePrefix"] = rootFolder;
                     // TODO: check if urdfGeom->filename is absolute or relative
                     std::string filename = rootFolder + "/" + urdfGeom->filename;
                     config["filename"] = filename;
